source/manual/alerts/RouterErrorRatioTooHigh.html.md

```diff
@@ -0,0 +1,35 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: Router error ratio too high
+parent: "/manual.html"
+layout: manual_layout
+section: Pagerduty alerts
+---
+
+You can find the router request error rates on this dashboard:
+
+- [5xx Router Requests][router-5x https://grafana.eks.production.govuk.digital/d/uBEWkijnz/router-request-rates-errors-durations?orgId=1 x-request-rates-grafana]
+
+You can also view the 500+503 error rates across all applications on this dashboard:
+
+- [5xx app Requests][app-5xx-error-rates-grafana]
+
+## description
+
+This alert will fire when the ratio of requests in an error state are above the threshold configured in
+this config [file][https://github.com/alphagov/govuk-helm-charts/blob/main/charts/monitoring-config/rules/router.yaml]
+
+## impact
+
+Router deals with requests to the rails applications which means that both publishers and/or end users could be seeing errors depending on which application is returning errors.
```

Comment:
Router is only used in request to frontend requests, not publishing applications. So a little bit confused about the distinction being made here about publishers and users possibly seeing errors.

```suggestion
Router deals with requests to frontend Rails applications which means that end users could be seeing errors depending on which application is returning errors.
```